<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="1"
            tests="1">

  <testsuite name="second"
    errors="1" failures="1" hostname="" id="" package="second" skipped="0"
    tests="1" time="10.34" timestamp="">
  
    <testcase name="Demo test cars two" classname="second" time="10.34" assertions="1">  <failure message="Testing if the URL is [0;33m&#39;http://localhost:3000/cars/0e1b2ea4-28a0-4983-88d0-36cfb71f4c22&#39;[0m in 5000ms - expected [0;32m&#34;is &#39;http://localhost:3000/cars/0e1b2ea4-28a0-4983-88d0-36cfb71f4c22&#39;&#34;[0m but got: [0;31m&#34;data:,&#34;[0m [0;90m(5132ms)[0m">    at Object.Demo test cars two (/home/vivanov/atScale/Rental-car-app/client/cars/tests/second.js:12:15)
    at &lt;anonymous&gt;</failure>

    
      <failure message="    at Object.Demo test cars two (/home/vivanov/atScale/Rental-car-app/client/cars/tests/second.js:12:15)">    at &lt;anonymous&gt;</failure>
      <error message="Error while running &#34;urlEquals&#34; command: Testing if the URL is [0;33m&#39;http://localhost:3000/cars/0e1b2ea4-28a0-4983-88d0-36cfb71f4c22&#39;[0m in 5000ms - expected [0;32m&#34;is &#39;http://localhost:3000/cars/0e1b2ea4-28a0-4983-88d0-36cfb71f4c22&#39;&#34;[0m but got: [0;31m&#34;data:,&#34;[0m [0;90m(5132ms)[0m" type="error">    at &lt;anonymous&gt;</error>
    
    </testcase>
  

  
    <system-err>
      [0;31m  NoSuchElementError: An error occurred while running .click() command on &lt;checkout&gt;: undefined; undefined
   {&#34;sessionId&#34;:&#34;5fbe05afc6fd329d4a21a1ac6f1aa172&#34;,&#34;status&#34;:0,&#34;value&#34;:[]}[0m
[0;90m       at &lt;anonymous&gt;
       at process._tickCallback (internal/process/next_tick.js:188:7)[0m
    </system-err>
  

  
  </testsuite>
</testsuites>
